tests/Feature/QueueWorkerTest.php

```diff
@@ -157,7 +157,7 @@ public function testRetryFailedJob(): void
         $output = $process_info['stdout'];
 
         $this->assertMatchesRegularExpression(
-            "~^.+failed.+{$failed_job_id}.+pushed\sback.+$~im",
+            "~^.+failed.+back.+$~im",
```

Comment:
Make your check stronger

Reply:
You mean, use some job identifier and check command output it to console ?

If yes, I literally can't do this, because:

For the lowest dependencies I can use this regex format: `"~^.+failed.+{$failed_job_id}.+pushed\sback.+$~im"` and everything is okay.

For the latest dependencies I catch an error: `Failed asserting that '   INFO  Pushing failed queue jobs back onto the queue.  ' matches PCRE pattern "~^.+failed.+88044191.+pushed\sback.+$~im".`, where I dont have any identifier in string expression.